packages/flutter/test/material/app_bar_test.dart

```diff
@@ -642,7 +623,7 @@ void main() {
         theme: ThemeData(platform: TargetPlatform.android),
         home: Scaffold(
           appBar: AppBar(
-            backwardsCompatibility: false,
+
```

Comment:
Why the empty line here?

Reply:
Oops, good catch. I'd forgotten to commit some local changes.